readme.md

```diff
@@ -25,8 +25,16 @@ jss source.css -p > source.jss
 ### API
 
 ```javascript
-import {cssToJss} from 'jss-cli'
-cssToJss(css, options)
+var css = [
+  '.ninja {',
+  '  visibility: hidden;',
+  '  color: black;',
+  '}'
+].join('\n')
+
+var css2jss = require('../lib/cssToJss.js')
+var jss = css2jss(css)
+console.log(JSON.stringify(jss, null, 2))
```

Comment:
I think because the whole js community is moving towards es6, we should write everything in es6. I did rewrite of jss in es6 already. We can start writing everything in es6. Examples are easy to write, no refactoring is needed.